tests/integration/features/conversation-1/ban.feature

```diff
@@ -63,7 +63,7 @@ Feature: conversation/ban
         | roomType | 3 |
         | roomName | room |
         And user "participant2" joins room "room" with 200 (v4)
-        And user "participant1" bans user "participant3" from room "room" with 404 (v1)
+        And user "participant1" bans user "participant3" from room "room" with 200 (v1)
```

Comment:
Just have a question on why this would be a response with 200, are we allowing the moderator to ban people who are not in the room?

Reply:
Yeah, as the UI has sending the ban parallel to the delete.
Also at some point we might add UI to ban IPs directly and they would also never be participants